doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -142,6 +142,10 @@ nrf9160 Samples
 
 * Updated:
 
+   * :ref:`at_monitor` sample:
```

Comment:
```suggestion
   * :ref:`at_monitor_sample` sample:
```
This should resolve the document build failure. My bad, didn't point it out earlier.